docs/src/virtual_machines/virtual_machine_usage.md

```diff
@@ -1,6 +1,6 @@
-## MDBCI usage
+# Virtual machine lifecycle
```

Comment:
The name in the navigation bar and the title of the page must be the same. Please check for all the pages.

Reply:
Also: what additional information does this page have compared with the quickstart? May be they should be combined into one.